src/main/kotlin/de/coasterfreak/fantasyfrontiers/data/model/items/Item.kt

```diff
@@ -0,0 +1,32 @@
+package de.coasterfreak.fantasyfrontiers.data.model.items
+class Item(private var _name: String, private var _translationKey: String, private var _maxItemStack: Int = 99) {
+
+    var name: String
+        get() = _name
+        set(value) {
+            _name = value
+        }
+
+    var translationKey: String
+        get() = _translationKey
+        set(value) {
+            _translationKey = value
+        }
+
+    var maxItemStack: Int
+        get() = _maxItemStack
+        set(value) {
+            _maxItemStack = value
+        }
```

Comment:
As the variables are all exposed, I would recommend to just expose them in the first place.
Also, i don't think we need to make those variables mutable.


```kotlin
data class Item(val name: String, val maxItemStack: Int = 99, val translationKey: String = "item.$name") {
```

This should do the job just fine

Reply:
fixed

Reply:
Okay i will do the required changes…thanks for the suggestion @CoasterFreakDE